src/planned-tests.html

```diff
@@ -50,6 +50,43 @@ <h3 class="govuk-body govuk-!-margin-bottom-6">
       <p class="govuk-body">
         If you get an alert, your phone or tablet may make a loud siren-like sound. You do not need to take any action.
       </p>
+
+      <h2 class="govuk-heading-m govuk-!-margin-bottom-2">
+        Ongoing
+      </h2>
+      <h3 class="govuk-body govuk-!-margin-bottom-6">
+        UK
+      </h3>
+      <p class="govuk-body">
+        Mobile phone networks in the UK are testing emergency alerts.
+      </p>
+      <p class="govuk-body">
+        You may get an alert if you have an Android phone or tablet.
+      </p>
+      <p class="govuk-body">
+        The alerts will say:
+      </p>
+      <div class="govuk-inset-text govuk-!-margin-top-2">
+        <p class="govuk-body">
+          This is a mobile network operator test of the Emergency Alerts service. You do not need to take any action. To find out more, search for gov.uk/alerts
+        </p>
+      </div>
+      <p class="govuk-body">
+        If you get an alert, your device may make a loud siren-like sound. You do not need to take any action.
+      </p>
+      <p class="govuk-body">
+        To opt out of these test alerts:
+      </p>
+      <ul class="govuk-list govuk-list--number">
+        <li>
+          Open your phone calling app.
+        </li>
+        <li>
+          Use the keypad to enter <span class="extra-letter-spacing govuk-!-font-weight-bold">*#*#2627#*#*</span>
+        </li>
+        <li>
+          Search your settings for ‘emergency alerts’ and turn off <b class="govuk-!-font-weight-bold">Test alerts</b>.
+        </li>
```

Comment:
We should probably link to the opt out page rather than duplicating the instructions in 2 places.

```suggestion
        You can <a href="/alerts/opt-out#mobile-network-tests">opt out of these test alerts</a>.
      </p>
```